### DISCUSSION TITLE
Get current scene as state attribute

### DISCUSSION THREAD

**Discussion by imbahzor** [mvdwetering/yamaha_ynca#49] [General]
Hello,

is it possible to get current scene selected on the amp as a state attribute?

would be great to have for automation where f.ex Scene 1 (movie) will turn off lights and close blinds.

i thought i could get this with the button press, but for me the button press only works as a command to send which scene to select and not to detect if you pressed that button the remote, not sure if that is intended or not.

i have the  rx-a1040 fyi

**Comment by mvdwetering** [comment]
That is not possible as the receiver does not have/provide that information.
It is only possible to activate a scene.

I don't know how you activate the Movie scene. If that is from Home Assistant that could also trigger the lights.

Otherwise you could maybe create an automation that triggers on changes on the receiver state and checks it the settings match the Movie scene. Not sure how complicated that would be.

**Comment by imbahzor** [comment]
thanks for the quick reply,

i activate the scene from the original remote, so i will probably then look into matching the scenes to specific attributes as you say and continue from there.

thanks for creating this integration it has helped me with lots of other automations
